crates/db_schema/src/source/person_block.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{schema::person_block, PersonBlockId, PersonId};
+use serde::Serialize;
+
+#[derive(Clone, Queryable, Associations, Identifiable, PartialEq, Debug, Serialize)]
+#[table_name = "person_block"]
+pub struct PersonBlock {
+  pub id: PersonBlockId,
+  pub person_id: PersonId,
+  pub recipient_id: PersonId,
```

Comment:
Maybe `target_id` would be a bit clearer?

Reply:
Either works, but recipient is clear enough, they're the recipient of a block.

Reply:
For me recipient is when something gets sent, but as the block doesnt get sent, I think target_id is clearer.